pkg/controller/volume/persistentvolume/pv_controller.go

```diff
@@ -1326,7 +1330,9 @@ func (ctrl *PersistentVolumeController) provisionClaimOperation(claimObj interfa
 		return
 	}
 
+	opComplete := util.OperationCompleteHook(plugin.GetPluginName(), "volume_provision")
```

Comment:
@jsafrane was that an oversight that `provisionClaimOperation` function does not return error, even if provisioning fails for some reason?